ReleaseTooling/Sources/FirebaseReleaser/Push.swift

```diff
@@ -42,9 +45,18 @@ enum Push {
       let command: String = {
         switch destination {
         case .cpdc:
-          return "pod repo push --skip-tests --use-json \(warningsOK) \(cpdcLocation) " +
+          var pushCommands = ""
+          if pod.isClosedSource {
+            // Push closed source pods to SpecsStaging to keep CI working.
+            pushCommands =
+              "pod repo push --skip-tests --use-json \(warningsOK) \(stagingLocation) " +
+              pod.skipImportValidation() + " \(pod.podspecName()) " +
+              "--sources=\(stagingRepo).git,https://cdn.cocoapods.org; "
+          }
+          pushCommands += "pod repo push --skip-tests --use-json \(warningsOK) \(cpdcLocation) " +
```

Comment:
Should this be in an `else` statement?

Reply:
No, it still needs to also push to cpdc

Reply:
Ah yep sorry just clicked for me, makes sense.